src/test/java/org/broadinstitute/hellbender/tools/spark/BaseRecalibratorSparkIntegrationTest.java

```diff
@@ -206,22 +221,22 @@ public void testBQSRBucket(BQSRTest params) throws IOException {
     @Test(description = "This is to test https://github.com/broadinstitute/hellbender/issues/322", groups = {"cloud"}, enabled = false)
     public void testPlottingWorkflow() throws IOException {
         final String resourceDir = getTestDataDir() + "/" + "BQSR" + "/";
-        final String GRCh37Ref = ReferenceAPISource.GRCH37_REF_ID; // that's the "full" version
+        final String chr2021Reference2bit = GCS_b37_CHR20_21_REFERENCE_2BIT;
```

Comment:
once we enable it, this is the reference we should use. I dont see a problem with updating disabled tests